tasks/assembly/task_shovill.wdl

```diff
@@ -6,21 +6,68 @@ task shovill_pe {
     File read2_cleaned
     String samplename
     String docker = "quay.io/staphb/shovill:1.1.0"
+
+    ## SHOVILL optional parameters
```

Comment:
@sage-wright can you expose these same params in the SE shovil task below?

Reply:
Yes, but I couldn't find the shovill_se docs to know if the options are different, since the main shovill docs say it's not applicable for SE?

Reply:
@kevinlibuit is that a robert fork?

Reply:
https://github.com/rpetit3/shovill

Reply:
Thanks Curtis! Optional params added for SE and just need to test those as well.